pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -408,6 +409,13 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 		}
 		// If we find a Pod that has not been created we create the Pod
 		if !isCreated(replicas[i]) {
+			if isStale, err := ssc.podControl.PodClaimIsStale(set, replicas[i]); err != nil {
+				return &status, err
+			} else if isStale {
+				// If a pod has a stale PVC, no more work can be done this round.
```

Comment:
That case of manual pod creation will be caught in UpdateStatefulPod.

A stale owner ref is where there's a ref to a different pod, meaning the race condition during a scale-down followed immediately by a scale back up before the PVC can get deleted. Since things may be in process of getting unmounted it seems best to just wait for everything to settle down.